src/hooks/useENSRegistrationCosts.ts

```diff
@@ -0,0 +1,122 @@
+import { useCallback, useMemo } from 'react';
+import { useQuery } from 'react-query';
+import { useAccountSettings } from '.';
+import { estimateENSRegistrationGasLimit } from '@rainbow-me/handlers/ens';
+import {
+  formatEstimatedNetworkFee,
+  formatRentPrice,
+  formatTotalRegistrationCost,
+} from '@rainbow-me/helpers/ens';
+import { Network } from '@rainbow-me/helpers/networkTypes';
+import { add, addDisplay, multiply } from '@rainbow-me/helpers/utilities';
+import { getEIP1559GasParams } from '@rainbow-me/redux/gas';
+import { ethereumUtils } from '@rainbow-me/utils';
+
+const secsInYear = 31536000;
```

Comment:
we should probably pass this to a const ens file

Reply:
yep, it's being used in a couple files already. A `timeUnits` file would be nice

Reply:
https://github.com/rainbow-me/rainbow/pull/2975